cfgov/unprocessed/js/organisms/VideoPlayer.js

```diff
@@ -0,0 +1,190 @@
+/* ==========================================================================
+   Base Video Player Class
+   ========================================================================== */
+
+// Required modules.
+import { checkDom, setInitFlag } from '../modules/util/atomic-helpers';
+import youTubeAPI from '../modules/youtube-api';
+import EventObserver from '../modules/util/EventObserver';
```

Comment:
Should be moved up a line to placate the https://eslint.org/docs/rules/sort-imports warning.

Reply:
Moved